plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterActive.php

```diff
@@ -30,16 +33,20 @@ protected function render( $attributes, $content, $block ) {
 		$active_filters = $block->context['activeFilters'];
 
 		$filter_context = array(
-			'items'  => $active_filters,
-			'parent' => $this->get_full_block_name(),
+			'items' => $active_filters,
 		);
 
 		$wrapper_attributes = array(
-			'data-wp-interactive'  => $this->get_full_block_name(),
 			'data-wp-key'          => wp_unique_prefixed_id( $this->get_full_block_name() ),
-			'data-wp-bind--hidden' => '!state.hasSelectedFilters',
-			/* translators:  {{label}} is the label of the active filter item. */
-			'data-wp-context'      => wp_json_encode( array( 'removeLabelTemplate' => __( 'Remove filter: {{label}}', 'woocommerce' ) ), JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
+			'data-wp-context'      => wp_json_encode(
+				array(
+					/* translators:  {{label}} is the label of the active filter item. */
+					'removeLabelTemplate' => __( 'Remove filter: {{label}}', 'woocommerce' ),
```

Comment:
If this translation does not vary, it is not necessary to include it in each context. It can be included only once using [`wp_interactivity_config`](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/api-reference/#wp_interactivity_config).

Reply:
`removeLabelTemplate` only belongs to Active Fitlers. `activeLabelTemplate` is different based on the filter type. So I think it's better in this case to use context.

Reply:
I see. `removeLabelTemplate` and `activeLabelTemplates` don't need to be reactive. I thought that'd make it easier for the dev mentally to get the label template only from context. But using context in terms of reactivity makes a lot of sense. It tells us the data is reactive without explicitly saying that.

Reply:
Addressed in f6fb54499c73

Reply:
It's not a big deal, but that'd be the correct way to do it in my opinion 🙂